backend/app/services/llm_judge.py

```
@@ -0,0 +1,150 @@
+            return "Неизвестно/Данные не предоставлены"
+
+        score_part = response.split(marker)[-1].strip()
+
+        if "Неизвестно" in score_part:
+            return "Неизвестно/Данные не предоставлены"
+
+        try:
+            score = int(score_part.split()[0])
+            if 1 <= score <= 10:
+                return score
+        except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)